main.py

```diff
@@ -51,10 +53,22 @@ def end_session(self, session_token: str) -> None:
 
     def get_customer(self, account_number: str) -> Union[Customer, None]:
         for customer in self.customers:
-            if customer.account.account_number == account_number:
-                return customer
+            for account in customer.accounts:
+                if account.account_number == account_number:
+                    return customer
         return None
 
+    def print_customers_info(self) -> None:
```

Comment:
printing class info in python can be done, by implementing python magic methods, its preferred way, that leverages OOP in python, and integrates with rest of python codebase